ovh/resource_domain_name_servers_test.go

```diff
@@ -0,0 +1,84 @@
+package ovh
+
+import (
+	"fmt"
+	"github.com/ovh/go-ovh/ovh"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func init() {
+	resource.AddTestSweepers("ovh_domain_name_servers", &resource.Sweeper{
+		Name: "ovh_domain_name_servers",
+		F:    testSweepOvhDomainNameServers,
+	})
+}
+
+func testSweepOvhDomainNameServers(region string) error {
```

Comment:
this func seems useless as it doesn't remove anything, or am I missing something ?